### ISSUE TITLE
[Bug]: Resource Creepers can spawn in Mushroom Fields biomes at night

### ISSUE THREAD

**Issue by streak0** [FTBTeam/FTB-Modpack-Issues#1087]
### Modpack

FTB Plexiglass Mountain

### Modpack version

1.1.1

### Log Files

https://paste.atlauncher.com/view/9aa5b2dc-ccff-49b1-b43f-220a0e31ed0a

### Describe the bug

As titled: Resource Creepers can naturally spawn in Mushroom Field biomes during night. Not sure if they also spawn underground; haven't done any mining in the biome yet. They seem to also spawn in much larger numbers than normal, which kinda makes sense given they are the only other mobs besides mooshrooms that can naturally spawn in the biome, as far as I am aware at least. Seems like it could get really broken really fast, balance wise, since you can just waltz around at night to farm them like normal hostile mobs in other biomes.

### Steps to reproduce

Find Mushroom Field biome, wait for night

### Expected behaviour

no mobs in the biome other than mooshrooms

### Additional information

I'm using ATLauncher and have Optifine installed, but after asking about it on the discord seems others have also noticed the behavior, presumably without optifine and an alternate launcher.

Not fully certain if it's actually a bug, but seems odd given that normally the biome is supposed to be free of mobs besides mooshrooms.

**Comment by Raidobw2** [comment]
I'll assume this is where you chose to make a home, so I'll post a quick solution if you want for now. I'll be able to ship that in the pack later on, possibly a 1.0.2 or 1.2.0.

First, open your modpack's folder where all the files are. Find the `config` folder and open it. Open the `incontrol` folder, then the `spawn.json` file with Notepad or Visual Studio Code.

Take this block of code (copy it):
```
{
        "dimension": "minecraft:overworld",
        "mod": "resourcefulcreepers",
        "biome": "minecraft:mushroom_fields",
        "result": "deny"
    },
```

And place it right after the first bracket `[`, like so (this can be done while the game is running btw):
![image](https://user-images.githubusercontent.com/17010632/178124212-6a773b18-966d-4dc0-94b1-0a12cea65fe6.png)
Make sure to save the file after!

Then in-game type `/incontrol reload`, and that's it. Resource Creepers should no longer be able to spawn in Mushroom Islands.

Note: I've not tested this in-game yet, but it should work :)

**Comment by Raidobw2** [comment]
I couldn't reproduce in-dev before adding that config, but I've added it regardless just in case

**Comment by Raidobw2** [comment]
Fixed in 1.2.0
